tests/workspace-files.feature

```diff
@@ -0,0 +1,39 @@
+# This file is part of REANA.
+# Copyright (C) 2023 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+Feature: Workspace files
+
+    As a researcher,
```

Comment:
We could perhaps simplify scenario to read:

```
As a researcher,
I want to make sure that the workspace produces expected files,
so that I can be sure that the workflow outputs are correct.
```

Because `gendata.C` and friends will be clarified in the scenarios below.  (So there should not be a need to put this detail at the top.)

Reply:
BTW reformulating with "So" in uppercase for consistency... I also changed a few words here and there.